### PR TITLE
Unified get_service behaviour

### PR DESCRIPTION
**Pull request by dmakhno**

### CODE REVIEW

File: qubell/api/private/organization.py [hunk 1]
```diff
@@ -314,24 +314,23 @@ def create_service(self, application=None, revision=None, environment=None, name
...
+    def get_service(self, id=None, name=None):
+        return self.get_instance(id, name)
 
     def list_services_json(self):
         return router.get_services(org_id=self.organizationId).json()
 
-    def get_or_create_service(self, id=None, application=None, revision=None, environment=None, name=None, parameters=None,
-                              type=None, destroyInterval=None):
-        """ Get by name or create service with given parameters"""
+    def service(self, id=None, application=None, revision=None, environment=None, name=None, parameters=None,
```
Comment (vasichkin) [inline qubell/api/private/organization.py]: Please,  leave get_or_create_service as deprecated. I'm not sure it's not used anywhere

File: qubell/api/private/organization.py [hunk 2]
```diff
@@ -314,24 +314,23 @@ def create_service(self, application=None, revision=None, environment=None, name
...
+                              type=None, destroyInterval=None, as_service_in=None):
+
         try:
             serv = self.get_instance(id=id, name=name)
-            if environment:
-                environment.add_service(serv)
+            serv.add_as_service(environments=as_service_in)
             return serv
         except exceptions.NotFoundError:
             return self.create_service(application, revision, environment, name, parameters, type)
```
Comment (vasichkin) [inline qubell/api/private/organization.py]: So, if add_as_service raises, we will create_service?

File: qubell/api/private/organization.py (hunk 1 above)
Comment (dmakhno) [inline qubell/api/private/organization.py]: done

File: qubell/api/private/organization.py (hunk 2 above)
Comment (dmakhno) [inline qubell/api/private/organization.py]: fixed

**Pull request merged**